sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredentialBuilder.java

```diff
@@ -8,21 +8,9 @@
  *
  * @see SharedTokenCacheCredential
  */
-public class SharedTokenCacheCredentialBuilder extends CredentialBuilderBase<ManagedIdentityCredentialBuilder> {
-    private String clientId;
+public class SharedTokenCacheCredentialBuilder extends AadCredentialBuilderBase<SharedTokenCacheCredentialBuilder> {
```

Comment:
Change in parent class can be backward incompatible but in this case since AadCredentialBuilderBase is also a type of CredentialBuilderBase, it might be okay but good to get @JonathanGiles to take a look.

Reply:
Yes this is to add `tenantId()` method to this builder through AadCredentialBuilderBase.